### PR TITLE
add annotate, freehand roi, and eraser, organize toolbar, highlight submenu tools

### PR DESCRIPTION
**Pull request by dlwire**
Handful of changes here. We can merge these into the sitka dev branch but using this temporary branch for now.

Toolbar
---
image

Draw submenu
---
image

Viewport submenu
---
image

Submenu tool active
---
image

Draw color changes (Cyan is the color while performing the act of drawing, Magenta is what it looks like afterwards)
---
image

Contributes to https://github.com/trustsitka/sitka/issues/3403

### CODE REVIEW

File: src/connectedComponents/ToolbarRow.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { commandsManager, extensionManager } from './../App.js';
 
 import ConnectedCineDialog from './ConnectedCineDialog';
 import ConnectedLayoutButton from './ConnectedLayoutButton';
-import ConnectedPluginSwitch from './ConnectedPluginSwitch.js';
+// import ConnectedPluginSwitch from './ConnectedPluginSwitch.js';
```
Comment (run-o) [inline src/connectedComponents/ToolbarRow.js:13]: should you remove these commented out lines or wanna keep it there for future reference?

File: src/connectedComponents/ToolbarRow.js [hunk 2]
```diff
@@ -128,7 +128,7 @@ class ToolbarRow extends Component {
           </div>
           {buttonComponents}
           <ConnectedLayoutButton />
-          <ConnectedPluginSwitch />
+          {/* <ConnectedPluginSwitch /> */}
```
Comment (run-o) [inline src/connectedComponents/ToolbarRow.js:131]: Same here, remove maybe?

**Review by run-o — APPROVED** [review]
I guess this lgtm

File: src/connectedComponents/ToolbarRow.js (hunk 1 above)
Comment (dlwire) [reply src/connectedComponents/ToolbarRow.js:13]: I was thinking keep them just to give us a little more memory about why they're gone next time around. When we merge in master we might see some changes around this.

I could go either way though.

Comment (run-o) [reply src/connectedComponents/ToolbarRow.js:13]: That's cool with me, just checking if it wasn't an omission.

**Review by Ggpsv — APPROVED** [review]

**Pull request merged**